components/vbout/vbout.app.mjs

```diff
@@ -376,46 +376,46 @@ export default {
 
       return response?.data;
     },
-    async moveContact({ ...args }) {
+    async moveContact(args) {
       return await this._makeRequest({
         path: "emailmarketing/movecontact",
         method: "POST",
-        args,
+        ...args,
       });
     },
-    async syncContact({ ...args }) {
+    async syncContact(args) {
       return await this._makeRequest({
         path: "emailmarketing/synccontact",
         method: "POST",
-        args,
+        ...args,
       });
     },
-    async updateContact({ ...args }) {
+    async updateContact(args) {
       return await this._makeRequest({
         path: "emailmarketing/editcontact",
         method: "POST",
-        args,
+        ...args,
       });
     },
-    async addActivity({ ...args }) {
+    async addActivity(args) {
       return await this._makeRequest({
         path: "emailmarketing/addactivity",
         method: "POST",
-        args,
+        ...args,
       });
     },
-    async createCampaign({ ...args }) {
+    async createCampaign(args) {
       return await this._makeRequest({
         path: "emailmarketing/addcampaign",
         method: "POST",
-        args,
+        ...args,
       });
     },
-    async createPost({ ...args }) {
+    async createPost(args) {
       return await this._makeRequest({
```

Comment:
You can even remove the `async/await` from here since what it matters is that `this._makeResquest` is returning just a `Promise`
```suggestion
    createPost(args) {
      return this._makeRequest({
```